### ISSUE TITLE
Analytics New Followers Summary heading is confusing

### ISSUE THREAD

**Issue by jsoref** [forem/forem#15664]
**Describe the bug**

The graph for new followers includes a range that goes below 0. The idea that it could do that just doesn't make sense.

There are two possible things that could be going on:
* The code doesn't know to clamp at 0 -- in which case the code should be changed to clamp at 0
* If one loses followers, the number could go negative, in which case the title could be better phrased, possibly "Follower Change Summary"

**To Reproduce**

1. Create an account
2. Use it over the span of a month
3. Visit the /dashboard/analytics endpoint
4. Switch to the Week view

**Expected behavior**

Either don't show negative values on the y-axis for a quantity that can't be negative, or use a title that indicates what it would mean for the quantity to be negative

**Screenshots**

![image](https://user-images.githubusercontent.com/2119212/144535076-5c604d16-90a8-4b97-97a6-e9a0cef73890.png)

**Desktop (please complete the following information):**

- OS, version: macOS Big Sur
- Browser, version: Chrome/96.0.4664.55

**Additional context**

**Comment by citizen428** [comment]
This is interesting, I don't see any negative numbers in mine. '

![Screen Shot 2021-12-06 at 11 37 56](https://user-images.githubusercontent.com/47985/144788077-39ea2083-ae3d-4ac4-bc5c-d0c26147b069.png)

Looking at `./app/javascript/analytics/dashboard.js` it seems like we reuse the same components for everything so it's imaginable that we're running in some default settings of the charting library here. At least that's where I'd start looking to track this problem down.

**Comment by jsoref** [comment]
Notably, you have followers. I don't. I'd bet it's an auto-scaling problem.

For some of these, it looks like you want `beginAtZero` per
https://www.chartjs.org/docs/master/axes/cartesian/linear.html#linear-axis-specific-options

**Comment by jsoref** [comment]
n.b. that's _assuming_ that "new lost followers" won't be reflected in the graph. It sounds like this is correct.

**Comment by citizen428** [comment]
> I'd bet it's an auto-scaling problem.

Yes, that's what I meant above, we reuse the same component, so I wouldn't be surprised if some defaults don't make sense in all contexts, e.g. using a float on the y-axis for new followers also doesn't seem like the best choice.

**Comment by barrymichaeldoyle** [comment]
I may be 2 years late to the party but yeah I still see nothing is done here, I assume lost followers will be reflected in the graph, in which case it is ok to show the negative axis.

The bigger problem is really the fact that there are fractional values on the chart.

**Comment by jsoref** [comment]
If it's change (this should be easily verified), the title should really be fixed – that's a trivial UI change.

**Comment by mikeydorje** [comment]
Closing as the followers chart referenced here no longer exists in the analytics dashboard. Feel free to open a new issue if you see a related problem in the current UI.
